Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Copyright 2019-present Intel Corporation
 
 # Stage bess-build: fetch BESS dependencies & pre-reqs
-FROM registry.aetherproject.org/sdcore/bess_build:251001 AS bess-build
+FROM registry.aetherproject.org/sdcore/bess_build:251001@sha256:b7d5a28a18bb51bbfcf4c36c52382797425fc94d8541af5d0690f17f6e9140f8 AS bess-build
```

Comment:
It seems it was already pinned ? registry.aetherproject.org/sdcore/bess_build:251001 . Isn't 251001 tag? Why we need to add SHA ?

Reply:
But are we reusing tag? Logically if tags are uniq and not reused then sha is not needed.